yarn.lock

```diff
@@ -0,0 +1,13123 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Remove yarn.lock I have added yar.lock in gitignore in latest code.It was hard to manage yarn.lock and package-lock.json at the same time.

Reply:
I am on it.🙌🏻

Reply:
Done @Abhishek-kumar09 .